src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -383,15 +388,44 @@ def create_uc_roles_cli(self, *, single_role=True, role_name="UC_ROLE", policy_n
                         f"{role_name}-{role_id}",
                         f"{policy_name}-{role_id}",
                         {s3_prefix},
-                        account_id=self._aws_account_id,
-                        kms_key=self._kms_key,
+                        self._aws_account_id,
+                        self._kms_key,
                     )
                 role_id += 1
 
+    def update_uc_role_trust_policy(self, role_name, external_id="0000"):
```

Comment:
let's document type parameters for public methods for parameters and return types. it'll help to avoid bugs.

Reply:
done